ribs/visualize.py

```diff
@@ -81,21 +85,23 @@ def _set_cbar(t, ax, cbar, cbar_kwargs):
     cbar_kwargs = {} if cbar_kwargs is None else cbar_kwargs
     if cbar == "auto":
         ax.figure.colorbar(t, ax=ax, **cbar_kwargs)
-    elif isinstance(cbar, axes.Axes):
+    elif isinstance(cbar, matplotlib.axes.Axes):
         cbar.figure.colorbar(t, ax=cbar, **cbar_kwargs)
 
 
-def grid_archive_heatmap(archive,
-                         ax=None,
-                         transpose_bcs=False,
-                         cmap="magma",
-                         square=None,
-                         aspect=None,
-                         vmin=None,
-                         vmax=None,
-                         cbar="auto",
-                         pcm_kwargs=None,
-                         cbar_kwargs=None):
+def grid_archive_heatmap(archive: GridArchive,
+                         ax: matplotlib.axes.Axes = None,
+                         transpose_bcs: bool = False,
+                         cmap: Union[str, list,
+                                     matplotlib.colors.Colormap] = "magma",
+                         square: bool = None,
+                         aspect: Union['auto', 'equal', float] = None,
+                         vmin: float = None,
+                         vmax: float = None,
+                         cbar: Union['auto', None,
+                                     matplotlib.axes.Axes] = "auto",
+                         pcm_kwargs: dict = None,
+                         cbar_kwargs: dict = None) -> GridArchive:
```

Comment:
This line is added to test if return type hints work. It does. 
- [ ] Remember to remove this line before merging.